src/models/test/group_test.ts

```diff
@@ -1,18 +1,14 @@
-import { expect } from "chai";
-import "mocha";
 import StudyGroup from "./../group";
 
 /**
  * Unit test suite for the study group model.
  */
-describe("Study group model", () => {
-  it("should be able to get its own ID", () => {
-    const group: StudyGroup = new StudyGroup(0, "groupName");
-    expect(group.id).to.equal(0);
-  });
+it("should be able to get its own ID", () => {
+  const group: StudyGroup = new StudyGroup(0, "groupName");

```

Comment:
Same as for the user (and this applies to all the others), should these tests be combined to be more efficient?

Reply:
Discussed offline, will resolve 👍